Moesif.NetCore.Test/Moesif.NetCore.Test.csproj

```diff
@@ -13,9 +13,13 @@
     <PackageReference Include="Moq" Version="4.12.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="2.2.0" />
+    <PackageReference Include="Moesif.Api" Version="2.0.4" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\Moesif.Middleware\Moesif.Middleware.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Moesif.Api" />

```

Comment:
What is this?

Reply:
I should remove it

Reply:
Ok.. Let me know after remove and running one test to confirm build and test passes.

Reply:
Tests run fine